src/components/FilterAutoCompleteInput/FilterAutoCompleteInput.tsx

```diff
@@ -32,18 +26,19 @@ export const FilterAutoCompleteInput: React.FC<FilterAutoCompleteInputProps> = (
     );
 
     return (
-        <StyledFormControl>
+        <FormControl>
             <FormControlInput
                 outline
                 autoFocus
                 placeholder={placeholder}
                 iconLeft={icon}
                 onChange={onChangeHandler}
+                width="100%"
```

Comment:
It would be better to use css for styles customizing. After some time it'll be much easier to find customizations.